styles/theme/_uswds-theme-custom-styles.scss

```diff
@@ -311,6 +311,7 @@ $theme-table-column-gap: 4;
 h1,
 .usa-prose h1 {
   @include u-text('semibold');
+  line-height: 1.15;
```

Comment:
USWDSv3 utilities would be better for this: https://designsystem.digital.gov/utilities/paragraph-styles/ 
in particular, the correct line-height for a given text node is actually dependent on the font it's in, and should be set with a SASS mixin (see "Utility Mixins" at https://designsystem.digital.gov/utilities/paragraph-styles/#utility-mixins-2)

example: 

`@include u-line-height('heading', 2)`

where `2` is one of the design tokens (I'm not sure if `2` is approximately 1.15 but find the one that's closest to what you're going for).